File: Python/Baekjoon/1546.py
import sys
input = sys.stdin.readline

N= int(input())
score_list = list(map(int, input().split()))
max_score = score_list[0]
total=0

for score in score_list:
    if score > max_score :
        max_score = score 

for i in score_list:
    sum_score = i /max_score*100
    total +=sum_score

        

print(total/N)

File: JavaScript/Baekjoon/1546.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs.readFileSync(filePath).toString().trim().split("\n");

const num = Number(input[0]);
const score = input[1].split(" ");

const max = Math.max(...score);
let sum = 0;

for (let i = 0; i < num; i++) {
  sum += (score[i] / max) * 100;
}

console.log(sum / num);
